Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: init
+init:
+	# ensure Homebrew is installed
+	which brew || /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+	# set up dev tools
+	brew bundle
+
+	# set up ruby environment
+	rbenv install --skip-existing
+	rbenv exec gem update bundler
+	rbenv exec bundle update
+
+	# install app dependencies via CocoaPods
+	rbenv exec bundle exec pod update
+
+	# ensure there's a .env file present
+	stat .env 2>/dev/null || echo "SENTRY_ORG=<your org slug>\nSENTRY_PROJECT=<your project slug>" > .env
```

Comment:
Do you think we should fail the build or at least output a message instructing to fill in those values in `.env`?
Because I don't remember if as of right now we're going to get a helpful error message if we leave those placeholder values

Reply:
Maybe, but that sounds like it should be its own changeset, as long as this PR is not degrading anything.

Reply:
fair! my bad

Reply:
all good! it's a good idea, i've just left this PR open too long already 😅